packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -486,13 +486,13 @@ $bpk-spacing-v2: true;
 @mixin bpk-checkbox {
   position: relative;
   display: inline-block;
-  padding-left: bpk-spacing-sm() * 7;
+  padding-left: bpk-spacing-lg() + bpk-spacing-sm();
 
   @include bpk-text;
   @include bpk-text--base;
 
   @include bpk-rtl {
-    padding-right: $bpk-spacing-base;
+    padding-right: bpk-spacing-lg() + bpk-spacing-sm();
```

Comment:
Missed this in the original PR! As you see the before is still using the old spacing style!